### PR TITLE
ReadMe hindi added

### PR DESCRIPTION
**Pull request by iamrahulmahato**
Hindi Translation readme added issue no #551

### CODE REVIEW

File: translations/README.hi.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+| पाठ संख्या |                             विषय                             |                  पाठ समूहन                  | सीखने के मकसद                                                                                                            |                                                             जुड़ा हुआ पाठ                                                              |                        लेखक                        |
+| :-----------: | :------------------------------------------------------------: | :-------------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------: |
+| 01 | मशीन लर्निंग का परिचय | [परिचय](1-Introduction/README.md)  | मशीन लर्निंग के पीछे की बुनियादी अवधारणाओं को जानें | [पाठ](1-Introduction/1-intro-to-ML/README.md)   | मुहम्मद |
```
Comment (jlooper) [inline translations/README.hi.md:82]: all of these links need to be turned to relative paths so they won't 404

File: translations/README.hi.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+---
+
+## शिक्षा शास्त्र
+
+इस पाठ्यक्रम का निर्माण करते समय हमने दो शैक्षणिक सिद्धांतों को चुना है: यह सुनिश्चित करना कि यह व्यावहारिक रूप से **परियोजना-आधारित** है और इसमें **लगातार प्रश्नोत्तरी** शामिल हैं। इसके अलावा, इस पाठ्यक्रम में एक समान **थीम** है जो इसे एकता प्रदान करता है।
+
+यह सुनिश्चित करके कि सामग्री परियोजनाओं के साथ संरेखित होती है, छात्रों के लिए प्रक्रिया को और अधिक आकर्षक बनाया जाता है और अवधारणाओं के प्रतिधारण को बढ़ाया जाएगा। इसके अलावा, कक्षा से पहले एक कम-दांव प्रश्नोत्तरी छात्र के विषय को सीखने के इरादे को निर्धारित करती है, जबकि कक्षा के बाद दूसरी प्रश्नोत्तरी आगे प्रतिधारण सुनिश्चित करती है। इस पाठ्यक्रम को लचीला और मजेदार बनाने के लिए डिज़ाइन किया गया था और इसे पूर्ण या आंशिक रूप से लिया जा सकता है। परियोजनाएं छोटी शुरू होती हैं और 12-सप्ताह के चक्र के अंत तक तेजी से जटिल हो जाती हैं। इस पाठ्यक्रम में एमएल के वास्तविक-विश्व अनुप्रयोगों पर एक पोस्टस्क्रिप्ट भी शामिल है, जिसका उपयोग अतिरिक्त क्रेडिट के रूप में या चर्चा के आधार के रूप में किया जा सकता है।
+
+> हमारी [आचार संहिता](CODE_OF_CONDUCT.md), [योगदान](CONTRIBUTING.md), और [अनुवाद](TRANSLATIONS.md) दिशानिर्देश खोजें। हम आपकी रचनात्मक प्रतिक्रिया का स्वागत करते हैं!
```
Comment (jlooper) [inline translations/README.hi.md:61]: this will 404, fix link

File: translations/README.hi.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+दुनिया भर में हमारे साथ यात्रा करें क्योंकि हम इन क्लासिक तकनीकों को दुनिया के कई क्षेत्रों के डेटा पर लागू करते हैं। प्रत्येक पाठ में पाठ से पहले और बाद में प्रश्नोत्तरी, पाठ को पूरा करने के लिए लिखित निर्देश, एक समाधान, एक असाइनमेंट, और बहुत कुछ शामिल हैं। हमारी परियोजना-आधारित शिक्षाशास्त्र आपको निर्माण करते समय सीखने की अनुमति देता है, जो नए कौशल को 'छड़ी' करने का एक सिद्ध तरीका है।
+
+**✍️ हमारे लेखकों ** जेन लूपर, स्टीफन हॉवेल, फ्रांसेस्का लाज़ेरी, टोमोमी इमुरा, कैसी ब्रेवियू, दिमित्री सोशनिकोव, क्रिस नोरिंग, अनिर्बान मुखर्जी, ओरनेला अल्टुनियन और एमी बॉयड को हार्दिक धन्यवाद।
+
+**🎨 हमारे चित्रकारों ** तोमोमी इमुरा, दासानी मदिपल्ली और जेन लूपर को भी धन्यवाद
```
Comment (jlooper) [inline translations/README.hi.md:21]: malformed markdown

**Review by jlooper — CHANGES_REQUESTED** [review]
Please fix all deadlinks and image paths

**Comment by jlooper** [comment]
Once this is fixed for 404s please send it to the discussion board for linguistic review, thanks

**Comment by iamrahulmahato** [comment]
@jlooper issue solved please review

**Comment by jlooper** [comment]
hi @iamrahulmahato thanks, let's ask for a linguistic review on the Discussion boards, please

File: translations/README.hi.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+| :-----------: | :------------------------------------------------------------: | :-------------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------: |
+| 01 | मशीन लर्निंग का परिचय | [परिचय](../1-Introduction/translations/README.it.md)  | मशीन लर्निंग के पीछे की बुनियादी अवधारणाओं को जानें | [पाठ](../1-Introduction/1-intro-to-ML/README.md)   | मुहम्मद |
+| 02 | मशीन लर्निंग का इतिहास | [परिचय](../1-Introduction/translations/README.it.md)  | इस क्षेत्र में अंतर्निहित इतिहास को जानें | [पाठ](../1-Introduction/2-history-of-ML/README.md)   | जेन और एमी |
+| 03 | निष्पक्षता और मशीन लर्निंग | [परिचय](../1-Introduction/translations/README.it.md)  | निष्पक्षता के आसपास कौन से महत्वपूर्ण दार्शनिक मुद्दे हैं जिन पर छात्रों को एमएल मॉडल बनाते और लागू करते समय विचार करना चाहिए? | [पाठ](../1-Introduction/3-fairness/README.md) | तोमोमी |
```
Comment (jlooper) [inline translations/README.hi.md:84]: not sure why the links here are going to the Italian translation?

File: translations/README.hi.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+| पाठ संख्या |                             विषय                             |                  पाठ समूहन                  | सीखने के मकसद                                                                                                            |                                                             जुड़ा हुआ पाठ                                                              |                        लेखक                        |
+| :-----------: | :------------------------------------------------------------: | :-------------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------: |
+| 01 | मशीन लर्निंग का परिचय | [परिचय](../1-Introduction/translations/README.it.md)  | मशीन लर्निंग के पीछे की बुनियादी अवधारणाओं को जानें | [पाठ](../1-Introduction/1-intro-to-ML/README.md)   | मुहम्मद |
```
Comment (jlooper) [inline translations/README.hi.md:82]: This is linking to the Italian translation

File: translations/README.hi.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+- एक पाठ समूह पूरा करने के बाद, [चर्चा बोर्ड](https://github.com/microsoft/ML-For-Beginners/discussions) पर जाएँ और उपयुक्त PAT रूब्रिक भरकर "ज़ोर से सीखें"। एक 'पीएटी' एक प्रगति आकलन उपकरण है जो एक रूब्रिक है जिसे आप अपने सीखने को आगे बढ़ाने के लिए भरते हैं। आप अन्य पीएटी पर भी प्रतिक्रिया कर सकते हैं ताकि हम एक साथ सीख सकें।
+
+> आगे के अध्ययन के लिए, हम इन [माइक्रोसॉफ्ट लर्न](https://docs.microsoft.com/en-us/users/jenlooper-2911/collections/k7o7tg1gp306q4?WT.mc_id=academic-15963-cxa) मॉड्यूल और सीखने के रास्तों का अनुसरण करने की सलाह देते हैं।.
+
+**शिक्षक**, हमारे पास [कुछ सुझाव शामिल हैं](../for-teachers.md) इस पाठ्यक्रम का उपयोग कैसे करें।
+---
+
+## टीम से मिलो
+
+[![प्रोमो वीडियो](../ml-for-beginners.png)](https://youtu.be/Tj1XWrDSYJU "Promo video")
```
Comment (jlooper) [inline translations/README.hi.md:48]: This should link to an animated gif

**Review by jlooper — CHANGES_REQUESTED** [review]
Please check the gif and links

**Comment by iamrahulmahato** [comment]
@jlooper done

**Review by jlooper — APPROVED** [review]

**Pull request merged**